console/console-init/ui/src/AppLayout.tsx

```diff
@@ -56,19 +63,48 @@ const AppLayout: React.FC = () => {
     []
   );
 
+  const [availableApps, setHasAvailableApps] = React.useState<
+    CrossNavApp[] | null
+  >(null);
+  const [showLogo, setShowLog] = React.useState(false);
+
+  if (!availableApps) {
+    getAvailableApps(
+      process.env.REACT_APP_RHMI_SERVER_URL
+        ? process.env.REACT_APP_RHMI_SERVER_URL
+        : getSolutionExplorerServer(),
+      undefined,
+      process.env.REACT_APP_RHMI_SERVER_URL ? "localhost:3006" : undefined,
+      ["3scale", "amqonline"],
+      !!process.env.REACT_APP_RHMI_SERVER_URL
+    ).then(apps => {
+      setHasAvailableApps(apps);
+      setShowLog(true);
+    });
+  }
+
   return (
     <ApolloProvider client={client}>
       <RootModal />
-      <Layout
```

Comment:
Is it okay to remove layout like this?

Reply:
Yes because their Layout component is what we need to override to add the CrossNavHeader. BUT I now see that I removed the avatar and toolbar, so thats why those settings weren't showing up :/ thanks for pointing that out.